environment.go

```diff
@@ -31,5 +33,46 @@ func (r *EnvironmentVariableResolver) Resolve(keys []string, _ ConfigMap) (val i
 	if len(r.Prefix) > 0 {
 		keysCopy = append([]string{r.Prefix}, keysCopy...)
 	}
-	return os.LookupEnv(strings.Join(toUpperStringSlice(keysCopy), "_"))
+
+	var translator KeyTranslator
+	if r.Translator == nil {
+		// If we weren't given a specific translator to use, then use the
+		// default translator.
+		translator = DefaultEnvironmentVariableKeyTranslator
```

Comment:
I feel like the default behavior should be to do nothing, no? Why does venom define what delimiters a user decides on?

Reply:
Well, the default behavior has always been to convert the keys to upper case and join them with underscores, so I think at a minimum that behavior should remain in the default translator, but you're probably right about the default behavior for mapping `-` to `_`